etc/adminhtml/system/adyen_security.xml

```diff
@@ -38,5 +38,29 @@
                 ]]>
             </comment>
         </field>
+        <field id="notification_hmac_key_test" translate="label" type="text" sortOrder="20" showInDefault="1"
+               showInWebsite="1" showInStore="1">
+            <label>Hmac key test</label>
+            <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+            <config_path>payment/adyen_abstract/notification_hmac_key_test</config_path>
+            <tooltip>
+                <![CDATA[
+                Set HMAC key at first on Adyen so new notifications are sent with the key. To learn more about these settings refer to
+                <a target="_blank" href="https://docs.adyen.com/classic-integration/hosted-payment-pages/hmac-signature-calculation#page-introduction">Adyen documentation</a>.
```

Comment:
this page is for HMAC signature for HPP skins, I think you want to point to the page for HMAC key for webhooks instead:
```suggestion
                <a target="_blank" href="https://docs.adyen.com/development-resources/webhooks/verify-hmac-signatures">Adyen documentation</a>.
```